pkg/crc/errors/error.go

```diff
@@ -0,0 +1,18 @@
+package errors
+
+import (
+	goerrors "errors"
+	"fmt"
+
+	"github.com/code-ready/crc/pkg/crc/logging"
+)
+
+func New(text string) error {
+	logging.Error("Error occured: %s", text)
```

Comment:
I am not sure about this, since we are logging the error here while creating it, and we usually pass up the error to the callers, the messages will be logged twice at least, once here while creating an error using `New/NewF` and also when the caller receives it and prints it.

Reply:
We do that on purpose as it can happen that in code you ignore and error with `_` or worse, but act upon it.

Reply:
OK, also this is logging in the `error` level so, it won't flood the console by default.